### DISCUSSION TITLE
Add support for Russian IBAN

### DISCUSSION THREAD

**Discussion by Elementenfresser** [Simplify/ibantools#343] [Ideas]
Hi! 

Thanks a lot for this ibantool, we love using it!

However, it seems that Russia is now also listed on the referenced [wikipedia page](https://en.wikipedia.org/wiki/International_Bank_Account_Number) and [IBAN registry](https://www.swift.com/resource/iban-registry-pdf)

It'd be great if you could find some time to extend your magic to cover Russia as well!

Thanks

**Comment by Simplify** [comment]
@Elementenfresser thanks for reporting this. 

Apparently Russia was added in May 2022 and I did not notice that. It's added now and new release is created.

**Reply by illyakurochkin** [reply]
Let's remove it again
